java/src/main/java/com/genexus/internet/HttpClientJavaLib.java

```diff
@@ -647,6 +666,7 @@ public void getHeader(String name, double[] value) {
 	public InputStream getInputStream() throws IOException {
 		if (response != null) {
 			this.setEntity();
+			streamsToClose.addElement(entity.getContent());
```

Comment:
Mm, I assume that being entity a stream calling getContent twice can be a problem. I would prefer to assign the stream first.

Something like:

   InputStream content = entity.getContent();
        streamsToClose.addElement(content);
        return content;

Reply:
Both suggestions are now implemented. Removed all trace of the getInputStream(URL) method and removed the double calling of getContent over a stream